.github/workflows/PR.yml

```diff
@@ -0,0 +1,35 @@
+name: PR
+on:
+        pull-request:
+                branches: [ master]
+jobs:
+        pull-test:
+                runs-on: ubuntu-latest
+                - steps:
+                        - uses: actions/checkout@v1
+                        - name: setup python3.9
+                          uses: actions/setup-python@v2
+                          with:
+                                  python-version: 3.9
+                        - name: Install dependencies
+                          run: |
+                                  python -m pip install --upgrade pip
+                                  pip install flake8 pytest
+                                  if [ -f 'requirements.txt ']; then pip install -r requirements.txt; fi
+                                  if [ -f 'tests_requirements.txt' ]; then pip install -r tests_requirements.txt; fi
+
+                        - name: Lint with flake8
+                          run: |
+                                  flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+                                  flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+                        - name: Configure AWS credentials for pytest
+                          uses: aws-actions/configure-aws-credentials@v1
+                          with:
+                                  aws-access-key-id: ${{ secrets.ACCESS_KEY_ID }}
+                                  aws-secret-access-key: ${{ secrets.SECRET_ACCESS_KEY }}
+                                  aws-region: us-east-2
+                        - name: Test with pytest
+                          run: |
+                                  python -m pytest
+
+
```

Comment:
Please check indentations:
It should be the same as 
https://github.com/redhat-performance/benchmark-runner/blob/main/.github/workflows/Func_Env_PR_Test_CI.yml